src/main/java/ch/njol/skript/variables/VariablesStorage.java

```diff
@@ -307,11 +324,10 @@ public final boolean load(SectionNode sectionNode) {
 	 *
 	 * @param backupInterval the backup interval.
 	 */
-	public void startBackupTask(Timespan backupInterval) {
+	public void startBackupTask(Timespan backupInterval, Boolean purge, Integer required, Integer remain) {
```

Comment:
should also probably use primitives here

Reply:
I can change it to int within the method, since I am not doing any checks here